pkg/prom/metrics.go

```diff
@@ -967,6 +970,24 @@ func NewMetricDescriptions(namespace string) *Descriptions {
 		nil,
 		nil,
 	)
+	d.VirtualVolumeTasks = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "cluster_volume_virtual_volume_task_count"),
+		"The total number of active virtual volume tasks in cluster",
+		nil,
+		nil,
+	)
+	d.BulkVolumeJobs = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "cluster_volume_bulk_volume_job_count"),
+		"The total number of active bulk volume jobs in cluster",
+		nil,
+		nil,
+	)
+	d.AsyncResults = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "cluster_volume_async_result_count"),
+		"The total number of active async results in cluster",
+		[]string{"status", "completed", "success"},
```

Comment:
Should this be `type` instead of `status`?

Reply:
fixed